config/betterquesting/DefaultQuests.json

```diff
@@ -1112,7 +1186,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "§oThis will be the first Project, or will it be?§r\n\nSome GregTech modpacks disable Charcoal furnace smelting, which forces the Coke Oven. Charcoal smelting is enabled here, making §dthis Multiblock optional in this modpack.§r\n\nYou can use it to make §aCoke from Coal§r, or §aCharcoal from Wood§r, in addition to §aCreosote as bonus§r output. Creosote is not very useful, except for a small amount of lubricant you may want past LV age.\n\n\nTo craft this multiblock, you will first need the brick form, which doesn\u0027t get consumed in recipes. You will need 117 clay, and 195 sand.\nCheck the JEI multiblock preview for the structure pattern.\nNote that the Hatch is optional and can be replaced with a bricks. The Hatch is used for automation purposes and can be placed on any centered face. You can also use multiple of them. You use the hatch to insert items or extract items and fluids.\n\n\n§bLore:§r The concept of Multiblock is very old in modded Minecraft. Back in 1.4.7, Railcraft introduced the Coke Oven, this was revolutionary at the time.",
+          "desc:8": "§oThis will be the first Project, or will it be?§r\n\nSome GregTech modpacks disable Charcoal furnace smelting, which forces the Coke Oven. Charcoal smelting is enabled here, making §dthis Multiblock optional in this modpack.§r\n\nYou can use it to make §aCoke from Coal§r, or §aCharcoal from Wood§r, in addition to §aCreosote as bonus§r output. Creosote is not very useful, except for a small amount of lubricant you may want past LV age.\n\n\nTo craft this multiblock, you will first need the brick form, which doesn\u0027t get consumed in recipes. You will need 117 clay, and 195 sand.\nCheck the JEI multiblock preview for the structure pattern.\n\n§9Note:§r Any Brick block can be replaced with a §9Coke Oven Hatch§r. That hatch allows for automation. It will auto-export Creosote into connected fluid containers, and it will also accept item insertion/extraction.\n\nIf you have no access to fluid voiding yet, but happen to have too much Creosote at hand, mine the Coke Oven block and replace it.\n\n\n§bLore:§r The concept of Multiblock is very old in modded Minecraft. Back in 1.4.7, Railcraft introduced the Coke Oven, this was revolutionary at the time.",
```

Comment:
```suggestion
          "desc:8": "§oThis will be your first Project... or will it?§r\n\nSome GregTech modpacks disable Charcoal furnace smelting, which forces the Coke Oven. Charcoal smelting is enabled here, making §dthis Multiblock optional in this modpack.§r\n\nYou can use it to make §aCoke from Coal§r, or §aCharcoal from Wood§r, in addition to §aCreosote as bonus§r output. Creosote is not very useful, except for a small amount of lubricant you may want past LV age.\n\n\nTo craft this multiblock, you will first need the brick form, which doesn\u0027t get consumed in recipes. You will need 117 clay, and 195 sand.\nCheck the JEI multiblock preview for the structure pattern.\n\n§9Note:§r Any Brick block can be replaced with a §9Coke Oven Hatch§r. That hatch allows for automation. It will auto-export Creosote into connected fluid containers, and it will also accept item insertion/extraction.\n\nIf you have no access to fluid voiding yet, but happen to have too much Creosote at hand, mine the Coke Oven block and replace it.\n\n\n§bLore:§r The concept of Multiblock is very old in modded Minecraft. Back in 1.4.7, Railcraft introduced the Coke Oven, this was revolutionary at the time.",
```

Also note that the coke oven is now basically, but technically still not required. The wooden large tank and primitive pump both need treated wood made from creosote now.